### PR TITLE
@eessex => [Node] Bump to v12

### PR DESCRIPTION
**Pull request by mzikherman**
Gonna use the PR Canary orb to test in Force.

### CODE REVIEW

File: src/node/node.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Orb Version 0.1.0
+# Orb Version 0.2.0
```
Comment (eessex) [inline src/node/node.yml]: I think this counts as a major version bump

Comment (zephraph) [reply src/node/node.yml]: I'm not sure that it will... let's just bump it to 1.0.0

Comment (zephraph) [reply src/node/node.yml]:
```suggestion
# Orb Version 1.0.0
```

**Comment by zephraph** [comment]
We can go ahead and merge this once the version is updated. We don't want to merge it in downstream repos yet though.

**Pull request merged**